Sky Atlas/Sky Atlas.html

```diff
@@ -12227,8 +12227,8 @@ <h1>Active Data</h1>
             || BLUE | <a
               href="https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf">https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf</a>
             <a href="https://etherscan.io/verifySig/19057">https://etherscan.io/verifySig/19057</a> | <a
-              href="https://etherscan.io/address/0xb18680092734394295d0591bb42f2bd3c184517e">https://etherscan.io/address/0xb18680092734394295d0591bb42f2bd3c184517e</a>
-            <a href="https://etherscan.io/verifySig/25260">https://etherscan.io/verifySig/25260</a> | <a
+              href="https://etherscan.io/address/0x73056E27D9cc08f963d41664C705392f305cFE7d">https://etherscan.io/address/0x73056E27D9cc08f963d41664C705392f305cFE7d</a>
```

Comment:
```suggestion
              href="https://etherscan.io/address/0xe1FF7ECf038900dd14A1E7E3a8CCF23f71f6A1c2">https://etherscan.io/address/0xe1FF7ECf038900dd14A1E7E3a8CCF23f71f6A1c2</a>
```
It seems the delegate made a mistake and didn’t enter the correct address for the contract.